packages/material-ui-docs/package.json

```diff
@@ -35,9 +35,9 @@
   },
   "peerDependencies": {
     "@material-ui/core": "^5.0.0-alpha.1",
-    "@types/react": "^16.8.6",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0"
+    "@types/react": "^16.8.6 || ^17.0.0",
```

Comment:
These aren't released yet. If `npm` and `yarn` don't complain on a fresh install we should get this in as well. `@types/react@17` is already proposed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48971